templates/scenarios/js/non-sso-tab/appPackage/manifest.json.tpl

```diff
@@ -6,9 +6,9 @@
     "packageName": "com.microsoft.teams.extension",
     "developer": {
         "name": "Teams App, Inc.",
-        "websiteUrl": "${{TAB_ENDPOINT}}",
-        "privacyUrl": "${{TAB_ENDPOINT}}/privacy",
-        "termsOfUseUrl": "${{TAB_ENDPOINT}}/termsofuse"
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com/termofuse",
```

Comment:
These two values seem to be reversed.

Reply:
Good Catch! These two values were copied from the default-bot template, and they were wrong from the beginning.